lib/src/view/home/home_tab_screen.dart

```diff
@@ -749,20 +749,33 @@ class _GamePreview<T> extends StatelessWidget {
 }
 
 class _RelationButton extends ConsumerWidget {
-  const _RelationButton();
+  const _RelationButton(this.wasOnline);
+
+  final bool wasOnline;
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     return AppBarIconButton(
-      icon: const Icon(Icons.people),
+      icon: Icon(
```

Comment:
You shouldn't change the button color. In flutter you disable a button by passing a `null` callback to the `onPressed` parameter. Flutter will disable and grey out the button automatically then.